src/test/resources/munit/orders-model.raml

```diff
@@ -15,7 +15,7 @@ types:
         (odata.key): false
         maxLength: 10
       ShipAddress:
-        type: nil | string
+        type: string | nil
```

Comment:
Is there any difference in AMF 5 with this?

Reply:
my bad